components/PressHeading.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react'
+
+interface Props {
+  headingData : {
+    title : string,
+    subtitle: string,
+  }
+}
+const PressHeading:FC<Props> = ({headingData}) => {
+  const {title, subtitle } = headingData;
+  return (
+    <div className='py-[4rem] tablet:py-[8rem]'>
```

Comment:
these rems need to be replaced with tailwinds classes. i.e. `py-2` or whatever the closest. 

There was a tweet that describes why this is important recently. https://twitter.com/t3dotgg/status/1574906239993028610